### PR TITLE
golangci-lint: remove unused settings, sort linters, and enable dupwords

### PR DESCRIPTION
**Pull request by thaJeztah**
Noticed in one of the forks that there was a dupwords issue 😂

### CODE REVIEW

File: .golangci.yml [hunk 1]
```diff
@@ -1,27 +1,18 @@
 linters:
   enable:
-    - staticcheck
-    - unconvert
+    - bodyclose
+    - dupword # Checks for duplicate words in the source code
```
Comment (milosgajdos) [inline .golangci.yml:4]: TIL 😄

**Review by Jamstah — APPROVED** [review]

**Review by milosgajdos — APPROVED** [review]
Let's wait for CI, but LGTM

File: .golangci.yml (hunk 1 above)
Comment (thaJeztah) [reply .golangci.yml:4]: About the linter or about what it does? 😂  Yeah, comment may be a bit redundant; I took it from the containerd PR where I spotted that they fixed the issue

Comment (milosgajdos) [reply .golangci.yml:4]: about the linter - I never used this one before, so wasnt aware of it. Super useful!

**Pull request merged**